Tests/PhpunitTest.php

```diff
@@ -1,10 +1,10 @@
-<?php
-namespace Kitpages\DataGridBundle\Tests;
-use PHPUnit\Framework\TestCase;
-class PhpunitTest extends TestCase
-{
-    public function testPhpunit()
-    {
-        $this->assertTrue(true);
-    }
-}
+<?php
```

Comment:
It is a bit strange. I can't understand the diff in that file (idem for all tests files)

Reply:
Hi. Its changes show because the test folder was deleted after flex update, and created again. No changes in tests.